test/integration/BUILD

```diff
@@ -10,6 +10,12 @@ load(
 
 envoy_package()
 
+native.genrule(
+    name = "hotrestart_test_disabled",
+    outs = ["hotrestart_disabled"],
+    cmd = "touch $@",
+)
```

Comment:
I can't figure out how to do this with tags.

The closest I've gotten is to rewrite `envoy_sh_test` to not access srcs in a way that prevents `select` from working. I'll push it in a bit.